test cases/nasm/1 configure file/meson.build

```diff
@@ -1,10 +1,6 @@
 project('nasm config file', 'c')
 
-if host_machine.cpu_family() == 'x86' and host_machine.system() == 'windows'
-  asm_format = 'win32'
-elif host_machine.cpu_family() == 'x86_64' and host_machine.system() == 'windows'
-  asm_format = 'win64'
-elif host_machine.cpu_family() == 'x86' and host_machine.system() == 'linux'
+if host_machine.cpu_family() == 'x86' and host_machine.system() == 'linux'
```

Comment:
Would be nice to make it work on Windows too, maybe just select a different asm source file as cross platform assembly is not really possible. It is pretty much the same source file in the 2nd nasm unit test.

Reply:
> as cross platform assembly is not really possible

In `3 nasm only` I've made a DLL, dylib and SO out of cross platform "assembly".

(Though if you want a runnable executable, you'd be right indeed. I'll add the example later today.)